### PR TITLE
non-software licenses page

### PR DESCRIPTION
**Pull request by mlinksva**
building on https://github.com/github/choosealicense.com/pull/307

non-software rather than non-code because software is eating the
    world, the main topic is open source software, and I find
    software/non-software slighly less trying than code/non-code if
    only because it isn't a
    [thing](http://www.adamhyde.net/perhaps-stop-using-non-code-whatayarekin/).

tweak text of page, also addressing #261 and #313

### CODE REVIEW

File: non-software.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+title: Non-Software Licenses
+layout: default
+permalink: /non-software/
+---
+
+Open source software licenses can be used for non-software works, and often are the best choice. This is particularly the case when the works in question can be edited and versioned as source, e.g., **[open source hardware](http://www.oshwa.org/definition/) designs**. Choose an [open source software license](/licenses/).
+
+[CC0-1.0](/licenses/cc0-1.0/), [CC-BY-4.0](/licenses/cc-by-4.0/), and [CC-BY-SA-4.0](/licenses/cc-by-sa-4.0/) are [open](http://opendefinition.org) licenses used for **non-software material ranging from datasets to videos**. Note that Creative Commons licenses should [not be used for software](https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software.).
+
+Any open source software license or open Creative Commons license is applicable to **software documentation**. If you use different licenses for your software and its documentation, you may wish to take care that the licenses are compatible, or that source code examples in the documentation is also licensed under the software license.
```
Comment (benbalter) [inline non-software.md]: > you may wish to take care that the licenses are compatible, or that source code examples in the documentation is also licensed under the software license.

How do I do that?

File: non-software.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+title: Non-Software Licenses
+layout: default
+permalink: /non-software/
+---
+
+Open source software licenses can be used for non-software works, and often are the best choice. This is particularly the case when the works in question can be edited and versioned as source, e.g., **[open source hardware](http://www.oshwa.org/definition/) designs**. Choose an [open source software license](/licenses/).
+
+[CC0-1.0](/licenses/cc0-1.0/), [CC-BY-4.0](/licenses/cc-by-4.0/), and [CC-BY-SA-4.0](/licenses/cc-by-sa-4.0/) are [open](http://opendefinition.org) licenses used for **non-software material ranging from datasets to videos**. Note that Creative Commons licenses should [not be used for software](https://creativecommons.org/faq/#can-i-apply-a-creative-commons-license-to-software.).
```
Comment (benbalter) [inline non-software.md]: I thought CC0 was okay for software?

**Comment by benbalter** [comment]
Nice. I feel like this is throwing a lot of information at the user, and not necessarily _demystifying_ the process. 

Two suggestions:
- Headings and organization to break things into logic groups and make things easier to skim / ignore non-applicable sections. After re-reading things, I think you're doing this with the bold text, but I'd front-load the subject matter / make it explicit.
- Instead of just stating facts, explain _how_ the user could use that fact, or more to the point, what they should do. Instead of "SIL accounts for this", even as simple as "Use a font-specific license, like the SIL". 

I'm torn, as to be helpful, we almost have to be prescriptive, but don't take that tone elsewhere in the site. Perhaps we can make it more like a decision tree?

File: non-software.md (hunk 1 above)
Comment (mlinksva) [inline non-software.md]: Heh. I think I'll just remove the bit about compatibility.

File: non-software.md (hunk 2 above)
Comment (mlinksva) [inline non-software.md]: I think it's marginal and wouldn't recommend it for a large project due to patent non-grant. But I'll tweak the language.

**Comment by mlinksva** [comment]
@benbalter thanks for the feedback, I'll try to address all in next commit. I was indeed trying to be non-prescriptive.

**Comment by benbalter** [comment]
>  I was indeed trying to be non-prescriptive.

Yeah, I'm torn and am not sure the best answer here. Either way, very glad you're doing this.

**Comment by MakerTobey** [comment]
I am a bit lost in the process. Does this mean that Open Source Hardware Licences, most prominently and widely used the CERN OHL will not become available on GitHub? It would be a highly desired addition in the choosealicence tool by many of the hardware developers active on GitHub.

**Comment by mlinksva** [comment]
@MakerTobey feel free to open an issue or make a PR about adding CERN OHL or any other license. The process is documented at https://github.com/github/choosealicense.com/blob/gh-pages/CONTRIBUTING.md#adding-a-license ... note there are a few criteria for inclusion. If an open hardware license meets all of the criteria but has not been approved by FSF or OSI or OD, we can look into adding an equivalent open hardware license approver to the criteria.

**Pull request merged**